backend/apps/slack/models/entity_channel.py

```
@@ -0,0 +1,21 @@
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from .conversation import Conversation
+
+class EntityChannel(models.Model):
```

Ruff (0.12.2)

Missing docstring in public class

(D101)